src/components/molecules/LoginComponent/LoginSection.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import Header from "../Header";
+import Image from "next/image";
+import BannerImage from "@/../../public/images/banner.png";
+import CopyrightInfo from "@/components/atoms/CopyrightInfo";
+import Link from "next/link";
+import styles from "./LoginSection.module.scss";
+import { MdOutlineAccessTimeFilled } from "react-icons/md";
+import { FaPhoneAlt } from "react-icons/fa";
+
+const LoginSection: React.FC = () => {
+  return (
+    <div className={styles.container}>
+      <Header />
+      <div className={styles.banner}>
```

Comment:
It would be better if we create a Banner component separately because a Banner is used on our other pages.
You can give image and text props. Could you resolve it?

Reply:
Okay I'll do it, should I create it in molecules or in atoms??